docs/guides/continuous-integration/circleci.mdx

```diff
@@ -99,72 +92,44 @@ Check out the full <Icon name="github" inline="true" callout="RWA Circle CI conf
 
 ## Additional Examples
 
-#### Example `.circleci/config.yml` v2 config file
+#### Component Testing Example
 
 ```yaml
-version: 2
-jobs:
-  build:
-    docker:
-      - image: cypress/base:14.16.0
-        environment:
-          ## this enables colors in the output
-          TERM: xterm
-    working_directory: ~/app
-    steps:
-      - checkout
-      - restore_cache:
-          keys:
-            - v1-deps-{{ .Branch }}-{{ checksum "package.json" }}
-            - v1-deps-{{ .Branch }}
-            - v1-deps
-      - run:
-          name: Install Dependencies
-          command: npm ci
-      - save_cache:
-          key: v1-deps-{{ .Branch }}-{{ checksum "package.json" }}
-          # cache NPM modules and the folder with the Cypress binary
-          paths:
-            - ~/.npm
-            - ~/.cache
-      - run: $(npm bin)/cypress run --record --key <record_key>
+version: 2.1
+orbs:
+  cypress: cypress-io/cypress@3
+workflows:
+  test:
+    jobs:
+      - cypress/run:
+          cypress-command: 'npx cypress run --component'
 ```
 
-#### Example `.circleci/config.yml` v2 config file with `yarn`
+#### Yarn Example
 
 ```yaml
-version: 2
-jobs:
-  build:
-    docker:
-      - image: cypress/base:14.16.0
-        environment:
-          ## this enables colors in the output
-          TERM: xterm
-    working_directory: ~/app
-    steps:
-      - checkout
-      - restore_cache:
-          keys:
-            - v1-deps-{{ .Branch }}-{{ checksum "package.json" }}
-            - v1-deps-{{ .Branch }}
-            - v1-deps
-      - run:
-          name: Install Dependencies
-          command: yarn install --frozen-lockfile
-      - save_cache:
-          key: v1-deps-{{ .Branch }}-{{ checksum "package.json" }}
-          paths:
-            - ~/.cache ## cache both yarn and Cypress!
-      - run: $(yarn bin)/cypress run --record --key <record_key>
+version: 2.1
+orbs:
+  cypress: cypress-io/cypress@3
+workflows:
+  test:
+    jobs:
+      - cypress/run:
+          package-manager: 'yarn'
+          start-command: 'yarn start'
 ```
 
-Find the complete CircleCI v2 example with caching and artifact upload in the
-[cypress-example-docker-circle](https://github.com/cypress-io/cypress-example-docker-circle)
-repo.
-
-#### RAM Disk
+#### Chrome Example
```

Comment:
I think any opportunity we have to clarify this bit will be valuable. Maybe a line under the Chrome Example header in smaller text does the trick. 

Cypress uses Electron by default to run your tests. `install-browsers` is used to install additional browsers to run your tests. This is only needed if you are passing the `--browser` flag in your `cypress-command`.

Reply:
Resolved in aade935bf6e28eb47483f83d95143e6d3dd8286d